run_kraken.py

```diff
@@ -31,6 +32,8 @@ def get_node_scenario_object(node_scenario):
         return aws_node_scenarios()
     elif node_scenario['cloud_type'] == 'gcp':
         return gcp_node_scenarios()
+    elif node_scenario['cloud_type'] == 'openstackcloud':
```

Comment:
Could we do ` 'openstack' in node_scenario['cloud_type']` instead, just to make the cloud type a little shorter